code/lib/core-server/src/utils/server-address.ts

```diff
@@ -10,7 +10,7 @@ export function getServerAddresses(
   initialPath?: string
 ) {
   const address = new URL(`${proto}://localhost:${port}/`);
-  const networkAddress = new URL(`${proto}://${host || ip.address()}:${port}/`);
+  const networkAddress = new URL(`${proto}://${host || internalIpV4Sync()}:${port}/`);
```

Comment:
First, we must change `getServerAddresses` to an async function. Then some code like this should work:

```suggestion
  const internalIP = await import('internal-ip');
  const networkAddress = new URL(`${proto}://${host || internalIP.internalIpV4Sync()}:${port}/`);
```